tests/unit_tests/data_steward/utils/participant_summary_requests_test.py

```diff
@@ -43,10 +42,11 @@ def setUp(self):
         self.fake_hpo = 'foo_hpo'
         self.destination_table = 'bar_dataset._deactivated_participants'
 
+        self.fake_token = 'ya29.12345'
```

Comment:
does the `fake_token` need to fit a particular format?  if not, we could make this even more blatantly fake.  what do you think?